app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -23,14 +23,30 @@ export const CONTACTS_NAVLINKS = [
   },
 ];
 
+export const ROLE = [
+  { value: "owner", label: "Owner" },
+  { value: "admin", label: "Admin" },
+  { value: "member", label: "Member" },
+];
```

Comment:
Create an util called `buildSelectOptions` and build the values based on the labels. Values can be kept as the camelCase versions of `label`.

Reply:
Yedhin I might need your help here. I had already tried out this option. But the problem I had while using `buildSelectOptions` was that eslint was raising the following error. 
<img width="512" alt="Screenshot 2022-09-23 at 10 35 21 PM" src="https://user-images.githubusercontent.com/56035039/192014782-538ffd5c-ab98-49c3-ad2f-ed76ab88b4c6.png">
Is there a way to avoid a dependency cycle. I thought of including `buildSelectOptions` in the same file (constants.js) but since its a util it will lose its value. Can you suggest a way if possible?